mqtt/mqttd/src/tracing/mod.rs

```diff
@@ -10,5 +10,6 @@ mod generic;
 #[cfg(all(not(feature = "edgehub"), feature = "generic"))]
 pub use generic::init;
 
-mod format;
-use format::Format;
+// TODO fix this
+// mod format;
+// use format::Format;
```

Comment:
What needs to be fixed here exactly?

Reply:
there are some changes in public API of tracing and I have a branch ready with changes: https://github.com/dmolokanov/iotedge/compare/tokio1-upgrade...dmolokanov:tracing-edgehub?expand=1